webpack.config.js

```diff
@@ -139,18 +148,24 @@ module.exports = function makeWebpackConfig () {
    * Reference: https://github.com/postcss/autoprefixer-core
    * Add vendor prefixes to your css
    */
-  config.postcss = [
-    autoprefixer({
-      browsers: ['last 2 version']
-    })
-  ];
+   // NOTE: This is now handled in the `postcss.config.js`
+   //       webpack2 has some issues, making the config file necessary
```

Comment:
First and formost, `webpack 2` does not allow "custom" config properties (can only use webpack native properties).   

So, the `postcss.config.js` replaces the use of 

```
...
   config.postcss()
...
```